frontend/packages/shipwright-plugin/src/types.ts

```diff
@@ -6,7 +6,17 @@ import { IBuild as IBuildV1Beta1 } from '@kubernetes-models/shipwright/shipwrigh
 import { IBuildRun as IBuildRunV1Beta1 } from '@kubernetes-models/shipwright/shipwright.io/v1beta1/BuildRun';
 import { IBuildStrategy as IBuildStrategyV1Beta1 } from '@kubernetes-models/shipwright/shipwright.io/v1beta1/BuildStrategy';
 import { IClusterBuildStrategy as IClusterBuildStrategyV1Beta1 } from '@kubernetes-models/shipwright/shipwright.io/v1beta1/ClusterBuildStrategy';
-import { K8sResourceCondition } from '@console/internal/module/k8s';
+import {
+  TektonResource,
+  TektonResultsRun,
+  TektonTaskSpec,
+} from '@console/dev-console/src/types/coreTekton';
+import { PipelineTaskParam, PipelineTaskRef } from '@console/dev-console/src/types/pipeline';
```

Comment:
This breaks the cycle:

```suggestion
import type { PipelineTaskParam, PipelineTaskRef } from '@console/dev-console/src/types/pipeline';
```

We should also be using `import type` more when the imports are type only